packages/core/src/web/app/constants/tutorial-constants.tsx

```diff
@@ -386,7 +386,9 @@ const INTERFACE_TUTORIAL: ITutorial = {
       },
       hintCircle: {
         height: 36,
-        right: calculateRight(192, RightRef.PATH_PREVIEW_BTN),
+        get right(): number {
+          return calculateRight(192, RightRef.PATH_PREVIEW_BTN);
+        },
```

Comment:
This should use `get` because `RightRef.PATH_PREVIEW_BTN` need to get workarea dynamically.
Other ones did, but this one didn't.